modules/streams/src/main/scala/dev/profunktor/redis4cats/interpreter/pubsub/Subscriber.scala

```diff
@@ -18,6 +18,8 @@ package dev.profunktor.redis4cats.interpreter.pubsub
 
 import cats.effect.{ ConcurrentEffect, ContextShift, Sync }
 import cats.effect.concurrent.Ref
+import cats.effect.implicits._
+import cats.implicits._
```

Comment:
This is not going to compile due to the `import cats.syntax.all._` below, needs to be removed...

Reply:
Sorry, I removed that, but I forgot to add that to my commit